roles/edge-image-builder/templates/config-eib.yaml.j2

```diff
@@ -0,0 +1,11 @@
+apiVersion: 1.0
+image:
+  imageType: raw
```

Comment:
We need to add `arch: x86_64` since https://github.com/suse-edge/edge-image-builder/pull/81 landed

Reply:
you're right! Done